src/App.tsx

```diff
@@ -0,0 +1,187 @@
+import titleImage from './assets/title.svg';
+import titleImageSM from './assets/titleSM.svg';
+
+const cardShift = -100;
+
+const useStyles = makeStyles((theme: Theme) =>
+    createStyles({
+        root: {
+            display: 'flex',
+            flexDirection: 'column',
+            height: '100vh',
+        },
+        content: {
+            flex: '1',
+        },
+        ellipseBackground: {
+            background: `url(${ellipseBackgroundImage}) ${theme.palette.primary.main}`,
+            backgroundSize: 'cover',
+            clipPath: 'ellipse(70% 200px at 50% 50px)',
+            width: '100%',
+            height: 250,
+        },
+        SimpleCardContainer: {
+            margin: '0 auto',
+            justifyContent: 'center',
+            position: 'relative',
+            top: cardShift,
+            display: 'flex',
+        },
+        title: {
+            backgroundImage: `url("${titleImage}")`,
+            backgroundSize: 'contain',
+            backgroundRepeat: 'no-repeat',
+            backgroundPosition: 'center',
+            position: 'absolute',
+            top: theme.spacing(4),
+            left: '50%',
+            transform: 'translateX(-50%)',
+            maxWidth: 500,
+            width: '90%',
+            height: 120,
+        },
+        section: {
+            backgroundColor: theme.palette.background.default,
+            padding: `${theme.spacing(3)}px 0`,
+            marginTop: cardShift + theme.spacing(2),
+        },
+        sectionContentContainer: {
+            maxWidth: 600,
+            margin: '0 auto',
+            padding: theme.spacing(1),
+        },
+        sectionTitle: {
+            textAlign: 'center',
+            marginBottom: theme.spacing(1),
+            fontWeight: 'bold',
+        },
+        sectionButtonContainer: {
+            display: 'flex',
+            justifyContent: 'center',
+        },
+        sectionButton: {
+            margin: `${theme.spacing(2)}px ${theme.spacing(1)}px`,
+        },
+        sectionButtonXS: {
+            margin: `${theme.spacing(1)}px 0`,
+        },
+        footer: {
+            backgroundColor: Colors.black[500],
+            padding: theme.spacing(2),
+            color: Colors.white[50],
+            flexShrink: 0,
+            textAlign: 'center',
+        },
+        FAB: {
+            position: 'fixed',
+            right: theme.spacing(3),
+            bottom: theme.spacing(3),
+            boxShadow: theme.shadows[10],
+        },
+    })
+);
+
+function App(): JSX.Element {
+    const theme = useTheme();
+    const classes = useStyles(theme);
+    const matchesSM = useMediaQuery(theme.breakpoints.down('sm'));
+    const matchesXS = useMediaQuery(theme.breakpoints.down('xs'));
+    return (
+        <div className={classes.root}>
+            <div className={classes.content}>
+                <div className={classes.ellipseBackground} />
+                <div
```

Comment:
Seems like we could get rid of some of these wrappers if we put the background image on body